api/ssi-bridge/src/services/ssi-service.ts

```diff
@@ -203,31 +213,37 @@ export class SsiService {
 		}
 	}
 
-	async generateIdentity(): Promise<{ account: Identity.Account; doc: Identity.Document; key: Identity.KeyPair }> {
+	async generateIdentity(): Promise<{
+		doc: Identity.Document;
+		signingKeys: Identity.KeyPair;
+		encryptionKeys: Identity.KeyPair;
+	}> {
 		try {
-			const builder = this.getAccountBuilder();
-			const keyPair = new KeyPair(KeyType.Ed25519);
-			const account = await builder.createIdentity({ privateKey: keyPair.private() });
+			const signingKeyPair = new KeyPair(KeyType.Ed25519);
+			const document = new Document(signingKeyPair, this.getConfig(false).network.name());
+
+			// Add encryption keys and capabilities to Identity
+			const encryptionKeyPair = new KeyPair(KeyType.X25519);
+			const encryptionMethod = new Identity.VerificationMethod(
+				document.id(),
+				encryptionKeyPair.type(),
+				encryptionKeyPair.public(),
+				'kex-0'
```

Comment:
could you please make this for now a const var defined at top of the method... we might make this later an enum